shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -861,10 +861,30 @@ - (void)dispatchTouches:(NSSet*)touches
   }
 
   const CGFloat scale = [UIScreen mainScreen].scale;
-  auto packet = std::make_unique<flutter::PointerDataPacket>(touches.count);
+
+  bool will_drop_hover = NO;
+
+  if (@available(iOS 13.4, *)) {
+    if ([UIFocusSystem focusSystemForEnvironment:self.view] == nil) {
```

Comment:
This API seems to be available since iOS 12? Maybe the version check is too restrictive?

Reply:
The mouse cursor API is only available on iOS 13.4. We should not remove the pointer if it isn't present, even though this API used is available earlier.